README.md

```diff
@@ -1,6 +1,15 @@
 # flotta-scale-tests
 The purpose of this project is to test the scalability and performance of [project-flotta](https://github.com/project-flotta/flotta-operator) using [JMeter](https://jmeter.apache.org).
 
+## Install OCP SNO cluster
+Clone https://gitlab.cee.redhat.com/jgil/simple-openshift-cluster-provisioner
```

Comment:
This repo isn't accessible outside. Instead, a user can use the underlying project https://github.com/openshift/assisted-test-infra to provision a cluster